app/jobs/get_usps_proofing_results_job.rb

```diff
@@ -163,15 +163,14 @@ def handle_bad_request_error(err, enrollment)
 
     if response_message == IPP_INCOMPLETE_ERROR_MESSAGE
       # Customer has not been to post office for IPP
-      enrollment_outcomes[:enrollments_in_progress] += 1
-      analytics(user: enrollment.user).
-        idv_in_person_usps_proofing_results_job_enrollment_incomplete(
-          **enrollment_analytics_attributes(enrollment, complete: false),
-          response_message: response_message,
-        )
-      enrollment.update(status_check_completed_at: Time.zone.now)
+      handle_incomplete_status_update(enrollment, response_message)
     elsif response_message&.match(IPP_EXPIRED_ERROR_MESSAGE)
-      handle_expired_status_update(enrollment, err.response, response_message)
+      # If we're blocking expirations, treat this enrollment as incomplete
+      if IdentityConfig.store.in_person_stop_expiring_enrollments.blank?
```

Comment:
just wondering - do we want to track at all which enrollments received a grace period ? Not sure if we'd want to do a look back and see the progress of enrollments that should have expired this week but didn't.

Reply:
@svalexander looks like the response_message is the same as one that has been expired: "More than 30 days have passed since opt-in to IPP". So we could look for this event "GetUspsProofingResultsJob: Enrollment incomplete" and the expired message, but yeah, maybe just adding another field could make that easier.

Reply:
Yeah, my thought was we could get a count with the query Jack described (or we could calculate expiration dates in prod console).